src/commands/setAppWizardContextDefault.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { Uri, workspace, WorkspaceConfiguration } from 'vscode';
+import { IAppServiceWizardContext, javaUtils, LinuxRuntimes, WebsiteOS } from 'vscode-azureappservice';
+import { LocationListStep } from 'vscode-azureextensionui';
+import { configurationSettings, extensionPrefix } from '../constants';
+
+export async function setAppWizardContextDefault(wizardContext: IAppServiceWizardContext): Promise<void> {
```

Comment:
This isn't really a command so I'm not sure it fits in this folder. Can we move this file closer to where it's actually used?